care/facility/tests/test_asset_api.py

```diff
@@ -131,3 +130,29 @@ def test_asset_filter_in_use_by_consultation(self):
         response = self.client.get("/api/v1/asset/?in_use_by_consultation=false")
         self.assertEqual(response.status_code, status.HTTP_200_OK)
         self.assertEqual(len(response.data["results"]), 1)
+
+    def test_asset_filter_warranty_amc_end_of_validity(self):
+        asset1 = Asset.objects.create(
+            name="asset1",
+            current_location=self.asset_location,
+            warranty_amc_end_of_validity=datetime.now().date(),
```

Comment:
- Use the linter to correct the failing lint check.
- Do not directly use naive datetime to create objects that expect a timezone aware datetime object, wrap this around Django's `make_aware` method. Example: `make_aware(datetime.now().date())`
- Or instead, here you can directly use `django.utils.timezone.now` to get the present aware datetime.